wolfProvider/grpc/grpc-FIPS-v1.60.0-wolfprov.patch

```
@@ -0,0 +1,195 @@
++fvUCAwEAAaOCAQkwggEFMAkGA1UdEwQCMAAwCwYDVR0PBAQDAgXgME8GA1UdEQRI
++MEaCECoudGVzdC5nb29nbGUuZnKCGHdhdGVyem9vaS50ZXN0Lmdvb2dsZS5iZYIS
++Ki50ZXN0LnlvdXR1YmUuY29thwTAqAEDMB0GA1UdDgQWBBSK4kHF8vH7iWk2JdkZ
++n1M936kTZDB7BgNVHSMEdDByoVqkWDBWMQswCQYDVQQGEwJBVTETMBEGA1UECAwK
++U29tZS1TdGF0ZTEhMB8GA1UECgwYSW50ZXJuZXQgV2lkZ2l0cyBQdHkgTHRkMQ8w
++DQYDVQQDDAZ0ZXN0Y2GCFFqz9Fbx3Mviz+lLmDbYi/YAYQ+aMA0GCSqGSIb3DQEB
++CwUAA4IBAQADc4orBdIANU3fyvkzmYPqp9zIEzWOVOpKoWMihmYi9DdDSnPDwTE0
++Ba8C/b4xRP4dAcVN5AkzT/p7717Qkwd/u/AsW41TW4XXP0kKG1JI8MUyZboXLxsP
++resGiNiCNqRpTSiFaN7Dot2TlSrvU4+84sUUm+xWeOpZ4u7LFaZyYTHn0AUB+1Zu
++tMN3H7xTDvBwMlpIP1gJA9yCabdRhEs2Tma9VgDWzrAHak5Lw7/FsrBvLW+C7MmH
++C0IJfYWpmMvTaDzFK3MdRuRHFW7PyEioapDM9/UDahIq/2Cy7GxaL4U2r3qFr2aI
++mo0jyWthmHpODGkHx/mwFjG85U6aMn/A
+ -----END CERTIFICATE-----
+diff --git a/test/core/handshake/client_ssl.cc b/test/core/handshake/client_ssl.cc
+index 45e39c1517..33262dca19 100644
+--- a/test/core/handshake/client_ssl.cc
++++ b/test/core/handshake/client_ssl.cc
+@@ -216,20 +216,49 @@ static void server_thread(void* arg) {
+   if (!ctx) {
+     perror("Unable to create SSL context");
+     ERR_print_errors_fp(stderr);
++    // Check if we're in force-fail mode (WOLFPROV_FORCE_FAIL)
++    const char* force_fail = getenv("WOLFPROV_FORCE_FAIL");
++    if (force_fail != nullptr && strcmp(force_fail, "1") == 0) {
++      gpr_log(GPR_ERROR, "SSL context creation failed (expected with WOLFPROV_FORCE_FAIL)");
++      return;  // Exit gracefully instead of abort
++    }
+     abort();
+   }
+
+   // Load key pair.
+-  if (SSL_CTX_use_certificate_file(ctx, SSL_CERT_PATH, SSL_FILETYPE_PEM) < 0) {
+-    perror("Unable to use certificate file.");
++  // First try loading the private key
++  if (SSL_CTX_use_PrivateKey_file(ctx, SSL_KEY_PATH, SSL_FILETYPE_PEM) < 0) {
++    perror("Unable to use private key file.");
+     ERR_print_errors_fp(stderr);
++    const char* force_fail = getenv("WOLFPROV_FORCE_FAIL");
```

This seems to suppress a failure when WPFF is set. If so, why do we want that?

---

We want to not fail out with WPFF that's why I am suppressing it so we dont do the abort preemptively